presto-yarn-docs/src/main/sphinx/debugging.rst

```diff
@@ -0,0 +1,31 @@
+Debugging and Logging
+=====================
+
+Once the YARN application is launched, you can monitor the status at YARN ResourceManager WebUI. 
+
+A successfully launched application will be in ``RUNNING`` state. The YARN ApplicationMaster UI (eg: ``http://master:8088/cluster/app/application_<id>``) will show slider-appmaster, COORDINATOR and WORKER components and the associated containers running based on your configuration. You can also use Slider cli script to check [status](#status).
+
```

Comment:
the link to (#status) has to be updated, right?like -  status <#check-the-status>